docs/en/pact-reference.md

```diff
@@ -388,31 +388,42 @@ as a unit.
+
 #### Module declaration {#moduledeclaration}
 
-[Modules](#module) contain the API and data definitions for smart contracts. They are comprised of:
+[Modules](#module) contain an API implementation and data definitions for smart contracts.
+They are comprised of:
 
 - [functions](#defun)
 - [schema](#defschema) definitions
 - [table](#deftable) definitions
 - ["pact"](#defpact) special functions
-- [const](#defconst) values
+- [constant](#defconst) values
 
-When a module is declared, all references to native functions
-or definitions from other modules are resolved. Resolution failure results in transaction rollback.
+When a module is declared, all references to native functions or definitions from other modules are resolved. Resolution failure results in transaction rollback.
 
-Modules can be re-defined as controlled by their admin keyset. Module versioning is not supported,
-except by including a version sigil in the module name (e.g., "accounts-v1"). However,
-*module hashes* are a powerful feature for ensuring code safety. When a module is imported with
-[use](#use), the module hash can be specified, to tie code to a particular release.
+Modules can be re-defined as controlled by their admin keyset. Module versioning is not supported, except by including a version sigil in the module name (e.g., "accounts-v1"). However, *module hashes* are a powerful feature for ensuring code safety. When a module is imported with [use](#use), the module hash can be specified, to tie code to a particular release.
 
-As of Pact 2.2, `use` statements can be issued within a module declaration. This combined with
-module hashes provides a high level of assurance, as updated module code will fail to import
-if a dependent module has subsequently changed on the chain; this will also propagate changes
-to the loaded modules' hash, protecting downstream modules from inadvertent changes on update.
+As of Pact 2.2, `use` statements can be issued within a module declaration. This combined with module hashes provides a high level of assurance, as updated module code will fail to import if a dependent module has subsequently changed on the chain; this will also propagate changes to the loaded modules' hash, protecting downstream modules from inadvertent changes on update.
 
 Module names must be globally unique.
 
+#### Interface Declaration {#interfacedeclaration}
+
+[Interfaces](#interface) contain an API specification and data definitions for smart contracts.
+They are comprised of:
+
+- [function](#defun) specifications (i.e. function signatures)
+- [constant](#defconst) values
```

Comment:
```suggestion
- [constant](#defconst) values
- formal verification models
```
Add explicit mention of models

Reply:
If we add this here, we should add it to the modules and defun section as well. Deferring to @slpopejoy for a y/n on that - there may be reasoning I don't know about.

Reply:
Added